frontend/src/OneBuilding/index.jsx

```diff
@@ -74,9 +75,16 @@ function OneBuilding({
     return () => clearInterval(updateResourcesInterval);
   }, []);
 
+  const [isVisible, setVisible] = useState(true);
```

Comment:
This is unnecessary. Error page will disappear by itself if the error becomes undefined in the Redux.